seed/tests/test_analysis_pipelines.py

```diff
@@ -613,6 +614,9 @@ def _mock_request(file_, model_type):
 
         return _mock_request
 
+    # Skipping this test b/c of an unexpected error validating BuildingSync files
+    # See here for more info: https://github.com/SEED-platform/seed/pull/2901
+    @skip
```

Comment:
we just discovered that gbxml.org/schema is not longer available, so I imagine that BSync is failing on this step. We have contacted gbxml.